Makefile

```diff
@@ -17,8 +17,9 @@ test: venv
 test-nomock: venv
 	@${VENV_NAME}/bin/tox -p auto -- --nomock $(TOX_ARGS)
 
-ci: venv
-	@${VENV_NAME}/bin/python setup.py test -a "-n auto --cov=stripe"
+test-travis: venv
+	${VENV_NAME}/bin/python -m pip install -U tox-travis
```

Comment:
[`tox-travis`](https://github.com/tox-dev/tox-travis) is a helper module to ensure that tox will only run tests using the Python version from the current Travis VM.